helm/tigris-stack/values-local-redundant.yaml

```diff
@@ -0,0 +1,137 @@
+# Copyright 2022 Tigris Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+### FoundationDB Cluster related settings
+fdb-cluster:
+  usable_regions: 1
+
+  processCounts: 
+    storage: 5
+    # log: 5
+    # storage: 5
+    # ratekeeper: 3
+    # data_distributor: 3
+
+  redundancy_mode: triple
+
+  resources:
+    requests:
+      cpu: "200m"
+      memory: "256Mi"
+    limits:
+      cpu: "200m"
+      memory: "256Mi"
+
+  volumeClaim:
+    resources:
+      requests:
+        storage: "3Gi"
+
+  failureDetectionTimeSeconds: 300
+  replaceInstancesWhenResourcesChange: true
+  automationOptions:
+    replacements:
+      enabled: true
+
+### Tigris Search related settings
+tigris-search:
+  image: tigrisdata/typesense:latest
+
+  ### API port, servicing requests
+  api_port: 8108
+
+  ### Peering port, used between the nodes to communicate with each other
+  peering_port: 8107
+
+  ### How much storage to allocate
+  storage: "1Gi"
+
+  ### Number of replicas
+  replicas: 5
+
+  ### Resources to allocate
+  resources:
+    limits:
+      cpu: "100m"
+      memory: "256Mi"
+    requests:
+      cpu: "100m"
+      memory: "256Mi"
+
+### Tigris Server related settings
+tigris-server:
+  image: tigrisdata/tigris:1.0.0-beta.18
```

Comment:
What are your thoughts about using `latest` images here?

Reply:
Or perhaps bring it in sync with the shell script's settings and pass/override from there with Helm `--set`?

Reply:
For now, using latest, later I will implement configurable versions of each component.